src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
```

Comment:
Good job on implementing the count feature! But according to the coding standards, imported classes should always be stated explicitly!